templates/base.html.twig

```diff
@@ -11,6 +11,7 @@
         <link rel="stylesheet" href="{{ asset('yarn/bootstrap3/dist/css/bootstrap-theme.min.css') }}"/>
 
         <link rel="stylesheet" href="{{ asset('css/base.css') }}"/>
+        <link rel='stylesheet' href='{{ asset('css/fonts.css') }}'/>
```

Comment:
Does this order matter? In my mind, fonts.css should go before base.css, but I'm not sure if there's any real difference; obviously the fonts will get caught up in the cascade, but I wonder if there's any reason to put it before the other stylesheets?

Reply:
Base.css does (or at least did) include some font definitions from the old sass file and they were overriding the fonts.css definitions. I don't think the order matters, but that's entirely up to you.

Reply:
That makes sense. I'll go and fix those font declarations so they aren't warring in my next PR and then flip the pointers.